webui/src/store/modules/user.js

```diff
@@ -154,8 +154,15 @@ export default {
                             SubjectSvc.getUserPhoto(elem.id, (done) => { done(null, token);}, data => {
                                 // Optimization: convert to base64 here and not in the OrgTree component
```

Comment:
Also I'm not sure the comment is actual now

Reply:
It is in theory, but agree it is not really important - let's remove